app/classifier/index.cjsx

```diff
@@ -40,6 +41,7 @@ Classifier = React.createClass
     subject: null
     classification: null
     onLoad: Function.prototype
+    cacheClassification: new CacheClassification
```

Comment:
Thanks for the tip. It works if I set it to a variable first and then export it. I was getting an undefined error with the import into the classifier otherwise.